db/migrate/20220531133731_add_blinded_to_boxes.rb

```diff
@@ -0,0 +1,11 @@
+class AddBlindedToBoxes < ActiveRecord::Migration
+  def up
+    change_table :boxes do |t|
+      t.boolean :blinded, default: false, null: false
+    end
+  end
+
+  def down
+    remove_column :boxes, :blinded
+  end
```

Comment:
Totally marginal, but couldn't this be a single `change` method?
```suggestion
  def up
    add_column :boxes, :blinded, :boolean, default: false, null: false
  end
```

Reply:
You're right. I'm used to write custom up/down methods...